util/Xamarin.AndroidBinderator/Xamarin.AndroidBinderator/Engine.cs

```diff
@@ -33,10 +33,13 @@ public static async Task BinderateAsync (BindingConfig config)
 
 	static async Task ProcessConfig (BindingConfig config)
 	{
+		Console.WriteLine("Processing config...");
```

Comment:
More spaces before parentheses.  (Or just remove these `Console.WriteLines` if they were for debugging and shouldn't be committed.)